packages/ripple-nuxt-tide/modules/publication/pages/PrintAll.vue

```diff
@@ -87,11 +87,6 @@ export default {
   props: {
     sidebar: Boolean
   },
-  head () {
-    return {
-      title: this.publication.title
-    }
-  },
```

Comment:
I would say for all static pages, this is more straight forward and easy to understand(Nuxt default). Not really feel we need to change to use store. How do you think?